--- a/print-802_11.c	
+++ b/print-802_11.c	
@@ -2033,16 +2033,16 @@
 			printf("%2.1f Mb/s ", .5*u.u8);
 		break;
 	case IEEE80211_RADIOTAP_DBM_ANTSIGNAL:
-		printf("%ddB signal ", u.i8);
+		printf("%ddBM antsignal ", u.i8);
 		break;
 	case IEEE80211_RADIOTAP_DBM_ANTNOISE:
-		printf("%ddB noise ", u.i8);
+		printf("%ddBM antnoise ", u.i8);
 		break;
 	case IEEE80211_RADIOTAP_DB_ANTSIGNAL:
-		printf("%ddB signal ", u.u8);
+		printf("%ddB antsignal ", u.u8);
 		break;
 	case IEEE80211_RADIOTAP_DB_ANTNOISE:
-		printf("%ddB noise ", u.u8);
+		printf("%ddB antnoise ", u.u8);
 		break;
 	case IEEE80211_RADIOTAP_LOCK_QUALITY:
 		printf("%u sq ", u.u16);
